docs/UserGuide.md

```diff
@@ -212,41 +212,35 @@ Example 1 (Listing all medications): `liststock`
 Expected Output 1:
 
 ```
-+====+==============+========+==============+=============+====================================+
-| ID |     NAME     | PRICE  |   QUANTITY   | EXPIRY_DATE |     DESCRIPTION     | MAX_QUANTITY | 
-+====+==============+========+==============+=============+====================================+
-| 1  |   PANADOL    | $20.00 |      20      | 13-09-2021  |      HEADACHES      |     1000     | 
-+----+--------------+--------+--------------+-------------+------------------------------------+
-| 2  |   PANADOL    | $20.00 |      10      | 14-09-2021  |      HEADACHES      |     1000     | 
-|    |              |        | PENDING: 150 |             |                     |              | 
-+----+--------------+--------+--------------+-------------+------------------------------------+
-| 3  |   VICODIN    | $10.00 |      20      | 30-09-2021  |     SEVERE PAIN     |     500      | 
-|    |              |        | PENDING: 30  |             |                     |              | 
-+----+--------------+--------+--------------+-------------+------------------------------------+
-| 4  | SIMVASTATIN  | $20.00 |      25      | 10-10-2021  |   HIGH CHOLESTEROL  |     800      | 
-|    |              |        | PENDING: 20  |             |                     |              | 
-+----+--------------+--------+--------------+-------------+------------------------------------+
-| 5  |  LISINOPRIL  | $20.00 |      25      | 15-10-2021  |    HYPOTHYROIDISM   |     800      | 
-|    |              |        | PENDING: 200 |             |                     |              | 
-+----+--------------+--------+--------------+-------------+-------------------------------- ---+
-| 6  | AZITHROMYCIN | $20.00 |      35      | 15-10-2021  |      INFECTIONS     |     100      | 
-|    |              |        | PENDING: 100 |             |                     |              | 
-+----+--------------+--------+--------------+-------------+------------------------------------+
++====+==============+========+==============+=============+=============================+
```

Comment:
Perhaps you would want to edit the stock.txt file and then printing the list command to prevent formatting issue.